### ISSUE TITLE
Should this work with rspack and esm output (outputModule: true)

### ISSUE THREAD

**Issue by codingnuclei** [ryoppippi/unplugin-typia#337]
Hi 👋 

Thanks for providing this great plugin 😄 

I can see there is a rspack export `@ryoppippi/unplugin-typia/rspack`, but I have no luck using rspack and this plugin when creating ESM output.
https://rspack.dev/blog/announcing-0-2#esmsystem-format

It works if you DO NOT use the below:

```js
module.exports = {
  // …
  experiments: {
    outputModule: true,
  },
  output: {
    chunkFormat: 'module',
    chunkLoading: 'import',
    library: {
      type: 'module',
    },
  },
};
```

Any ideas?

**Comment by ryoppippi** [comment]
Could you give me a reproduction repository

**Comment by codingnuclei** [comment]
So as it turns out it has nothing to do with esm at all!

I narrowed it down to a point in my project where I used a ts paths to import a type i.e `@my-org/types`.

For some reason when used like this it fails, but if I change to relative pathing it works.

I tried replicating it in a fresh rspack project, but it all worked.

The project where it fails is a rather complex mono-repo so my only guess is that something is mis configured.

From what i can see unplugin-typia is working as expected 🥳 

Thanks
